File: samples/src/main/java/tk/zielony/carbonsamples/guidelines/ButtonsUsageActivity.java
package tk.zielony.carbonsamples.guidelines;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import carbon.widget.ProgressBar;
import carbon.widget.Toolbar;
import tk.zielony.carbonsamples.R;

public class ButtonsUsageActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_buttonsusage);

        Toolbar toolbar = findViewById(R.id.toolbar);
        toolbar.setIconVisible(true);

        ProgressBar progress = findViewById(R.id.progress);
        progress.setProgress(0.8f);
    }
}

File: samples/src/main/java/tk/zielony/carbonsamples/guidelines/ButtonsUsageActivity.kt
package tk.zielony.carbonsamples.guidelines

import android.os.Bundle
import kotlinx.android.synthetic.main.activity_buttonsusage.*
import tk.zielony.carbonsamples.SampleAnnotation
import tk.zielony.carbonsamples.R
import tk.zielony.carbonsamples.SamplesActivity

@SampleAnnotation(layoutId = R.layout.activity_buttonsusage, titleId = R.string.buttonsUsageActivity_title)
class ButtonsUsageActivity : SamplesActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        toolbar.setIconVisible(true)
        progress.progress = 0.8f
    }
}
